bam_ampliconclip.c

```diff
@@ -1113,6 +993,7 @@ int amplicon_clip_main(int argc, char **argv) {
             case 1012: param.rejects_file = optarg; break;
             case 1013: param.oa_tag = 1; break;
             case 1014: param.del_tag = 0; break;
+            case 1015: param.tol = atoi(optarg);  if (param.tol < 0) param.tol = 0; break;
```

Comment:
Perhaps it would be better to go back to the default (5), instead of 0.

Reply:
My thinking was that 0 was the nearest valid value to a negative number.  Going back to the default is reasonable too.  I should probably make it print out an error message when it changes the value.